### ISSUE TITLE
Alpha channels displayed in delay and with stutters on top of playing image sequences

### ISSUE THREAD

**Issue by wigglepixel** [EsotericSoftware/spine-editor#706]
Hi!

Please take a look at the video below. Something strange is going on while playing image sequences (PNG's):

Somehow either Spine is showing the alpha channel op top with a delay and some stutter, or this is an alpha channel filled with white. But either way; the viewport is having this weird issue, but the preview panel (playing) is not.

The scene is made up of multiple image sequences, so what you see is not just one sequence messed up, but ALL sequences messed up! (for instance the dress is 1 sequence, the head is one sequence, the fire etc.)

BTW I checked if there was something selected in the tree to cause something like a selection outline like this, but there wasn't. Switching to other objects in the tree didn't fix this.

Happily I created this screenrecording before restarting Spine, because after restarting the issue was gone. Not sure if it will return, but if so I'll let you know.

Using Spine 4.1.09 pro

https://user-images.githubusercontent.com/7702506/192367083-5a26429b-1b80-453a-8033-943721e63fb3.mp4

**Comment by NathanSweet** [comment]
It looks like you had [ghosting](http://esotericsoftware.com/spine-ghosting) enabled (`alt+H` in animate mode to show the Ghosting view).

**Comment by wigglepixel** [comment]
> It looks like you had [ghosting](http://esotericsoftware.com/spine-ghosting) enabled (`alt+H` in animate mode to show the Ghosting view).

At first when I saw this response by you I thought you were right and I didn't check before if I had accidentally turned on ghosting (eventhough I didn't use ghosting and didn't have the ghosting panel open). 

But as you can see in the video below it's not ghosting. This is something else. In the video below you can see that ghosting looks a lot different. Instead of the thing I saw, ghosting is moving way more and also fluently and with colors. That's not what I saw in the situation before (as you can see in the video in the opening post).

https://user-images.githubusercontent.com/7702506/192384037-c067f0c1-c8bc-49a9-b5bd-23b0106207f3.mp4

**Comment by NathanSweet** [comment]
Ghosting could be configured to look like the first video, with settings like this:

![](http://n4te.com/x/5443-1TFD.png)

If you didn't enable ghosting, then I don't know. From the video has to be something to do with ghosting, as you can see the pose is from the future in the animation and when the skeleton reaches that pose, the next future pose is shown.

We'll close this for now, but if you see it again or find a way to reproduce the problem, please post in this issue and we'll reopen.

**Comment by NathanSweet** [comment]
Open issues track work that we need to do. This issue is not being ignored, it still exists but is closed because there is no work for us to do. We need to be able to reproduce the problem to be able to fix it. The problem disappeared when Spine was restarted and is no longer bothersome. You are the only user to ever report this problem and you only saw it once. If you continue to see it or others see it and we can collect more information, then we will reopen this issue. Until then there is no work for us to do and so we don't need the clutter of an open issue.

I contend it does look like ghosting. Nothing else, even a bug, would show a pose from the future of the animation. I understand you didn't configure ghosting to look like that, but there could be some sort of bug that renders ghosting when it should not.

I noticed that if the dopesheet view is not visible when Spine starts that the `Keys` ghosting does not show or shows incorrectly. We have fixed that in 4.1.10.
